packages/react/src/SubNav/SubNav.features.stories.tsx

```diff
@@ -92,6 +92,98 @@ WithShadow.parameters = {
   layout: 'fullscreen',
 }
 
+export const SubHeading = args => (
+  <SubNav {...args}>
+    <SubNav.Heading href="#">AI</SubNav.Heading>
+    <SubNav.SubHeading href="#">Copilot</SubNav.SubHeading>
+    <SubNav.Link href="#">For business</SubNav.Link>
+    <SubNav.Link href="#" aria-current="page">
+      Extensions
+    </SubNav.Link>
+    <SubNav.Link href="#">Tutorials</SubNav.Link>
+    <SubNav.Link href="#">
+      What&apos;s new
+      <SubNav.SubMenu>
+        <SubNav.Link href="#">Sub-feature page one</SubNav.Link>
+        <SubNav.Link href="#">Sub-feature page two</SubNav.Link>
+        <SubNav.Link href="#">Sub-feature page three</SubNav.Link>
+        <SubNav.Link href="#">Sub-feature page four</SubNav.Link>
+      </SubNav.SubMenu>
+    </SubNav.Link>
+    <SubNav.Link href="#">Plans & pricing</SubNav.Link>
+    <SubNav.Action href="#">Call to action</SubNav.Action>
+  </SubNav>
+)
+SubHeading.parameters = {
+  layout: 'fullscreen',
+}
+
+export const SubHeadingNarrow = args => <SubHeading {...args} />
+SubHeadingNarrow.parameters = {
+  layout: 'fullscreen',
+  viewport: {
+    defaultViewport: 'iphonex',
+  },
+}
+
+export const SubHeadingNarrowOpen = args => <SubHeading {...args} />
+SubHeadingNarrowOpen.parameters = {
+  layout: 'fullscreen',
+  viewport: {
+    defaultViewport: 'iphonex',
+  },
+}
+SubHeadingNarrowOpen.play = async ({canvasElement}) => {
+  const canvas = within(canvasElement)
+  await userEvent.click(canvas.getByTestId('SubNav-root-button'))
```

Comment:
Really minor and appreciate this is more of a demo than a test, but it would be nice to use a `getByRole` in these cases, rather than adding more `getByTestId` uses to the codebase.

Reply:
Good suggestion. Will defer that for now given the urgency for this to ship.